src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ossirian.cpp

```diff
@@ -63,28 +69,28 @@ enum Events
     EVENT_STOMP             = 3
 };
 
-uint8 const NUM_CRYSTALS = 9;
+uint8 const NUM_CRYSTALS = 12;
 
-// You spin me right round, baby
-// right round like a record, baby
-// right round round round
 Position CrystalCoordinates[NUM_CRYSTALS] =
 {
-    { -9394.230469f, 1951.808594f, 85.97733f, 0.0f },
-    { -9357.931641f, 1930.596802f, 85.556198f, 0.0f },
-    { -9383.113281f, 2011.042725f, 85.556389f, 0.0f },
-    { -9243.36f, 1979.04f, 85.556f, 0.0f },
-    { -9281.68f, 1886.66f, 85.5558f, 0.0f },
-    { -9241.8f, 1806.39f, 85.5557f, 0.0f },
-    { -9366.78f, 1781.76f, 85.5561f, 0.0f },
-    { -9430.37f, 1786.86f, 85.557f, 0.0f },
-    { -9406.73f, 1863.13f, 85.5558f, 0.0f }
+    { -9188.4404296875f, 1940.2099609375f, 85.6390991210937f, 3.17650008201599f },
+    { -9244.41015625f, 1808.97998046875f, 85.6390991210937f, 5.63741016387939f },
+    { -9248.41015625f, 1974.82995605469f, 85.6390991210937f, 5.89920997619629f },
+    { -9282.080078125f, 1887.33996582031f, 85.6390991210937f, 2.00712990760803f },
+    { -9299.73046875f, 1748.44995117187f, 85.6390991210937f, 1.44861996173859f },
+    { -9357.8603515625f, 1929.07995605469f, 85.6390991210937f, 1.06465005874634f },
+    { -9367.169921875f, 1780.89001464844f, 85.6390991210937f, 1.90241003036499f },
+    { -9383.2900390625f, 2012.68005371094f, 85.6511001586914f, 2.93214988708496f },
+    { -9406.099609375f, 1862.38000488281f, 85.6390991210937f, 6.2308201789856f },,
```

Comment:
doesnt compile, you have two commas at the end here

Reply:
upsy, corrected.